packages/api/src/router/user.router.ts

```diff
@@ -66,6 +66,25 @@ export const userRouter = createTRPCRouter({
     })
   }),
 
+  /**
+    * get a user's friends
+    */
```

Comment:
this could be place in the friend router instead, it called the `friend` collection.

Reply:
Will update tonight. Thanks!